relay-node/benchmarks/src/reference_hardware.json

```diff
@@ -13,10 +13,10 @@
     },
     {
         "metric": "DiskSeqWrite",
-        "minimum": 536.40
+        "minimum": 220.0
```

Comment:
Why this big difference ?

Reply:
That's a great question, there is something weird indeed. The machine benchmarks on our reference machine were failing on these metric, so I set them accordingly. But this is probably unexpected and undesirable.
Maybe we should revert this change for this PR, and investigate further? cc @la10736